web/src/components/Plugin/locales/en-US.ts

```diff
@@ -21,6 +21,14 @@ export default {
   'component.step.select.pluginTemplate.select.option': 'Custom',
   'component.plugin.pluginTemplate.tip1': '1. When a route already have plugins field configured, the plugins in the plugin template will be merged into it.',
   'component.plugin.pluginTemplate.tip2': '2. The same plugin in the plugin template will override one in the plugins',
+  'component.plugin.authentication': 'Authentication',
+  'component.plugin.security': 'Security',
+  'component.plugin.traffic': 'Traffic Control',
+  'component.plugin.serverless': 'Serverless',
+  'component.plugin.observability': 'Tracing & Metrics & Logging',
+  'component.plugin.logging': 'Logging',
```

Comment:
Do we still need `logging`? or assign them to `observability` too?

Reply:
vote for observability